internal/value/value.go

```diff
@@ -0,0 +1,106 @@
+package value
```

Comment:
Should we add a comment to the tune of https://github.com/hashicorp/terraform-plugin-sdk/blob/main/internal/plans/objchange/normalize_obj.go#L18, given that we do not check the type of the cty value because it's not really necessary?

Reply:
Yep that makes sense 👍🏻 thanks for spotting this